roles/packages/files/test-mise-verification.sh

```
@@ -0,0 +1,147 @@
+    fi
+done
+
+echo ""
+echo "==> Binary launch check (every audited tool runs)"
+for tool in "${!EXPECTED_MAP[@]}"; do
+    bin="${BINARY_MAP[$tool]:-$tool}"
+    if "$bin" --version >/dev/null 2>&1 \
+        || "$bin" version --client >/dev/null 2>&1 \
+        || "$bin" version >/dev/null 2>&1 \
+        || "$bin" -v >/dev/null 2>&1; then
+        [ "$bin" != "$tool" ] && ok "$tool launches (binary: $bin)" || ok "$tool launches"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)